### DISCUSSION TITLE
Gemini Lake

### DISCUSSION THREAD

**Discussion by cyring** [cyring/CoreFreq#490] [General]
@N0tACyb0rg : Hello

Can you please show the N5000 Memory Controller output using latest _CoreFreq_ ?

I believe I have introduced a regression within Gemini Lake code path

**Comment by N0tACyb0rg** [comment]
I believe you are correct in that a regression has been introduced, sorry for the late reply.
This is the output from `corefreq-cli -M` on v1.97.2.
```
                           Goldmont Plus  [31F0]                           
Controller #0                                                    Disabled 
```

**Reply by cyring** [reply]
Thanks
I'm now remembering it was left as an [unsolved issue](https://github.com/cyring/CoreFreq/issues/389#issue-comment-box)
Perhaps if I have a chance to work with another Gemini Lake I might find the missing points.

**Comment by cyring** [comment]
Dev ceased
